tests/fv3_conf/cpld_control_run.IN

```diff
@@ -159,20 +158,56 @@ else
      fi
 
   else
-     # C48mx500 warm start from ICs
-     cp    ${FV3_IC}/*.nc ./INPUT
-     cp    ${MOM_IC}/MOM.res.nc ./INPUT
-     cp    ${ICE_IC}/* .
-     cp    @[INPUTDATA_ROOT]/CMEPS_IC/${ATMRES}mx${OCNRES}/${SYEAR}${SMONTH}${SDAY}${SHOUR}/* .
+     if [[ ${BMIC} == .true. ]]; then
+       #cp ${FV3_IC}/gfs*.nc ./INPUT
+       cp ${FV3_IC}/fv*.nc ./INPUT
+       cp ${FV3_IC}/sfc_data*.nc ./INPUT
+       cp ${FV3_IC}/phy_data*.nc ./INPUT
+       #cp ${FV3_IC}/coupler.res ./INPUT
+       #coupler.res fv_core.res.nc fv_core.res.tile[1-6].nc fv_srf_wnd.res.tile[1-6].nc fv_tracer.res.tile[1-6].nc phy_data.tile[1-6].nc fv3_increment.nc
+       cp ${MOM_IC}/MOM*.nc ./INPUT
+       cp ${MOM_IC}/mom6_increment.nc ./INPUT
+       cp ${MOM_IC}/interpolate_zgrid_26L.nc ./INPUT
+       cp ${ICE_IC}/cice*nc ./INPUT
+       ls -1 ./INPUT/cice*nc>ice.restart_file
+       cp ${FV3_IC}/*stoch*nc ./INPUT
+       cp ${FV3_IC}/ca_data*nc ./INPUT
+       cp ${FV3_IC}/ufs.cpld.cpl*nc .
+       ls -1 ufs.cpld.cpl*nc>rpointer.cpl
+       cp ${WAV_IC}/ufs.cpld.ww3.r.${RESTART_FILE_SUFFIX_SECS} .
+     else
+       # C48mx500 warm start from ICs
+       cp    ${FV3_IC}/*.nc ./INPUT
+       cp    ${MOM_IC}/MOM.res.nc ./INPUT
+       cp    ${ICE_IC}/* .
+       cp    @[INPUTDATA_ROOT]/CMEPS_IC/${ATMRES}mx${OCNRES}/${SYEAR}${SMONTH}${SDAY}${SHOUR}/* .
+     fi
   fi
 fi
 
 #inline post
 if [ $WRITE_DOPOST = .true. ]; then
-  cp    ${PATHRT}/parm/post_itag_gfs itag
-  cp    ${PATHRT}/parm/postxconfig-NT-gfs.txt postxconfig-NT.txt
-  cp    ${PATHRT}/parm/postxconfig-NT-gfs_FH00.txt postxconfig-NT_FH00.txt
-  cp    ${PATHRT}/parm/params_grib2_tbl_new params_grib2_tbl_new
+  if [[ ${BMIC} == .true. ]]; then
+    cp    ${PATHRT}/../FV3/upp/parm/gefs/post_itag_gefs128 itag
+    cp    ${PATHRT}/../FV3/upp/parm/gefs/postxconfig-NT-gefs.txt postxconfig-NT.txt
+    cp    ${PATHRT}/../FV3/upp/parm/gefs/postxconfig-NT-gefs-f00.txt postxconfig-NT_FH00.txt
+    cp    ${PATHRT}/parm/params_grib2_tbl_new params_grib2_tbl_new
```

Comment:
@NickSzapiro-NOAA You might change the path of  'params_grib2_tbl_new' from ${PATHRT}/../FV3/upp/parm/.

Reply:
@WenMeng-NOAA Sure. May I ask you:
1. What are the differences in postxconfig-NT-gefs-f00.txt, postxconfig-NT-gefs-aerosol.txt, & postxconfig-NT-gefs-chem.txt? Like when to use which
2. Would you like the ${BMIC} == .false. files just below this to come from upp as well, now or eventually?

Reply:
@NickSzapiro-NOAA 
1. The files postxconfig-NT-gefs-aerosol.txt, & postxconfig-NT-gefs-chem.txt are used by the model developers for their own testing. The postxconfig-NT-gefs-f00.txt and postxconfig-NT-gefs.txt have been used for GEFS V13 development. 
2. At this time, please keep the current settings for " ${BMIC} == .false.".